src/clj/puppetlabs/services/ca/certificate_authority_service.clj

```diff
@@ -27,9 +30,9 @@
              comidi/routes->handler)
          settings
          path
-         wrap-with-authorization-check
-         puppet-version)))
-   context)
+         (fn [request] (wrap-with-authorization-check request {:oid-map oid-mappings}))
```

Comment:
It looks like this is duplicated in 4 different places (here, admin service, legacy service, master service).  Any way we can DRY that up?  Spitballing here, but ... perhaps, instead of adding the `get-oid-mappings` service function, instead we could add a `auth-middleware-with-oid-mappings` service function which would encapsulate this.  Thoughts?  @nathanielksmith?